utils.py

```diff
@@ -16,7 +17,10 @@ def get_stored_flows(dpids: list = None, state: str = "installed"):
     if state:
         char = '&' if dpids else '/?'
         api_url += char+f'state={state}'
-    result = requests.get(api_url)
+    try:
+        result = requests.get(api_url, timeout=30)
+    except ConnectTimeout as exception:
```

Comment:
This is one step in the right direction, but notice that it's still not fully handled. If a `ConnectTimeout` happens you're handling will just log, and then it'll try to get a `result.json()` so it'll still crash, which isn't what we want for a reliable back-end code.

Here you have a failing dependency that isn't available, since `get_stored_flows` is always called in a request handler, then you can raise HTTP 424 (or better yet, just let a RetryError with tenacity be raised and then try catch to raise HTTP 424 where `get_stored_flows` is called), that way `mef_eline` and any other callers will be able to differentiate when it actually errored out and can't trust the response at the moment.

Also, notice that `ConnectTimeout` doesn't catch `ReadTimeout`, so you're supposed to catch both `requests.exceptions.Timeout` and `requests.exceptions.ConnectionError`. Also, let's make sure to have retries here with `tenacity` which is already a depedency that `kytos` ships with, you can decorate `get_stored_flows` with `@retry` following a similarish parametrization that [we have on our DB controllers](https://github.com/kytos-ng/flow_manager/blob/master/controllers/__init__.py#L25-L33)